keyvault/client/secrets/README.md

```diff
@@ -0,0 +1,337 @@
+```java
+try {
+    SecretClient.getSecret("deletedSecret")
+} catch (ResourceNotFoundException e) {
+    System.out.println(e.getMessage());
+}
+```
+### Logging [TODO]
+This SDK uses SLF4J logging library. The logging is enabled by default, just ensure your application/project takes dependency on SLF4J logging library.
+
+## Next steps
+Several KeyVault Java SDK samples are available to you in the SDK's GitHub repository. These samples provide example code for additional scenarios commonly encountered while working with Key Vault:
+
+### Hello World Samples
+* [HelloWorld.java][sample_helloWorld] - and [HelloWorldAsync.java][sample_helloWorldAsync] - Contains samples for following scenarios:
+    * Create a Secret
+    * Retrieve a Secret
+    * Update a Secret
+    * Delete a Secret
+
+### List Operations Samples
+* [ListOperations.java][sample_list] and [ListOperationsAsync.java][sample_listAsync] - Contains samples for following scenarios:
+    * Create a Secret
+    * List Secrets
+    * Create new version of existing secret.
+    * List versions of an existing secret.
+
+### Backup And Restore Operations Samples
+* [BackupAndRestoreOperations.java][sample_BackupRestore] and [BackupAndRestoreOperationsAsync.java][sample_BackupRestoreAsync] - Contains samples for following scenarios:
+    * Create a Secret
+    * Backup a Secret -- Write it to a file.
+    * Delete a secret
+    * Restore a secret
+
+### Managing Deleted Secrets Samples:
+* [ManagingDeletedSecrets.java][sample_ManageDeleted] and [ManagingDeletedSecretsAsync.java][sample_ManageDeletedAsync] - Contains samples for following scenarios:
+    * Create a Secret
+    * Delete a secret
+    * List deleted secrets
+    * Recover a deleted secret
+    * Purge Deleted secret
+    
+
+## Contributing
+This project welcomes contributions and suggestions. Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the Code of Conduct FAQ or contact opencode@microsoft.com with any additional questions or comments.
+
+<!-- LINKS -->
+[source_code]: https://github.com/Azure/azure-sdk-for-java/tree/master/keyvault/client/secrets/src
+[package]:https://mvnrepository.com/artifact/com.microsoft.azure/azure-keyvault
+[api_documentation]: not-valid-link
```

Comment:
Have you filed an issue on yourself to update this link?

Reply:
Created an issue. #4016